evm-contracts/src/v0.6/dev/AggregatorProxy.sol

```diff
@@ -44,41 +56,51 @@ contract AggregatorProxy is AggregatorInterface, AggregatorV3Interface, Owned {
     override
     returns (uint256 updatedAt)
   {
-    ( , , , updatedAt, ) = latestRoundData();
+    return AggregatorInterface(currentPhase.aggregator).latestTimestamp();
   }
 
   /**
    * @notice get past rounds answers
-   * @param _roundId the answer number to retrieve the answer for
+   * @param _requestId the answer number to retrieve the answer for
    * @dev deprecated. Use getRoundData instead.
    */
-  function getAnswer(uint256 _roundId)
+  function getAnswer(uint256 _requestId)
     public
     view
     virtual
     override
     returns (int256 answer)
   {
-    ( , answer, , , ) = getRoundData(_roundId);
+    (
+      uint256 requestRoundId, ,
+      address aggregator
+    ) = getRoundIdPhaseIdAndAggregator(_requestId);
```

Comment:
Dangling comma looks strange in this format, consider adding a comment and place it on a new line:
```suggestion
    (
      uint256 requestRoundId, 
      /* skip */,
      address aggregator
    ) = getRoundIdPhaseIdAndAggregator(_requestId);
```